src/V3AstNodeOther.h

```diff
@@ -704,17 +704,17 @@ class AstCUse final : public AstNode {
     // Parents:  NODEMODULE
 private:
     const string m_name;
-    const VUseType m_useType;  // What sort of use this is
+    const uint8_t m_useType;  // What sort of use this is
```

Comment:
We should use the type.  if you had to do this to get the logical operations against it working, then instead add a method in the VUseType enumeration and call it from the visitor.

Reply:
Methods increase verbosity when dealing with logical operations. I need to explicitely call the constructor of `VUseType` just to use a method. Eg.:

```cpp
VUseType::INT_INCLUDE | VUseType::INT_FWD_CLASS
```
vs.
```cpp
VUseType{VUseType::INT_INCLUDE}.or(VUseType::INT_FWD_CLASS)
```

But, consider, this:
```cpp
constexpr VUseType::en operator|(VUseType::en lhs, VUseType::en rhs) {
    return VUseType::en{lhs | rhs};
}
constexpr VUseType::en operator&(VUseType::en lhs, VUseType::en rhs) {
    return VUseType::en{lhs & rhs};
}
```
This would allow me to keep the type while allowing logical operations to be performed on flags.

Reply:
IMO adding the operators and keeping the types is cleaner.